05-JWT-Basics/starter/errors/bad-request.js

```diff
@@ -0,0 +1,15 @@
+const CustomAPIError = require("./custom-error");
+const { StatusCodes } = require("http-status-codes");
+
+
+class BadRequestError extends CustomAPIError {
+  constructor(message) {
+    super(message);
+    //this.statusCode = 400
+    this.statusCode = StatusCodes.BAD_REQUEST;
```

Comment:
make sure you study the popular status codes a little bit; it's handy to know what they are. These are the most common ones:

- 200
- 201
- 400
- 403
- 404
- 500